bundles/ru.arsysop.svn.connector/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,15 @@
+Manifest-Version: 1.0
+Automatic-Module-Name: ru.arsysop.svn.connector
+Bundle-ManifestVersion: 2
+Bundle-SymbolicName: ru.arsysop.svn.connector
+Bundle-Version: 0.1.0.qualifier
+Export-Package: ru.arsysop.svn.connector
+Require-Bundle: org.eclipse.core.runtime;bundle-version="3.18.0",
+ org.eclipse.osgi.services;bundle-version="3.8.0";resolution:=optional,
```

Comment:
This bundle dependency should be avoided, please prefer `Import-Package` instead if you need it.